explorer/src/Pos/Explorer/ExtraContext.hs

```diff
@@ -10,12 +10,12 @@ module Pos.Explorer.ExtraContext
 
     , HasGenesisRedeemAddressInfo (..)
     , GenesisRedeemAddressInfo
-    -- ^ Genesis address info
+    -- * Genesis address info
```

Comment:
Why a '-- *' here instead of '--'?

Reply:
`-- *` Is acceptable and creates a contents table. I can also replace it with `-- `or put the description above the functions with a `-- |`. In my opinion, the best for the html generated by haddock is the latter and the best for someone reading the source code is the former.
@jcmincke which should I prefer?